router/pump.go

```diff
@@ -140,7 +140,7 @@ func (p *LogsPump) Run() error {
 		debug("pump.Run() event:", normalID(event.ID), event.Status)
 		switch event.Status {
 		case "start", "restart":
-			go p.pumpLogs(event, true, inactivityTimeout)
+			go p.pumpLogs(event, false, inactivityTimeout)
```

Comment:
Perhaps we should only set this to `false` on restart and have a `true` case on start. Thoughts?